### DISCUSSION TITLE
How to create a new label in GitHub mobile for issues

### DISCUSSION THREAD

**Discussion by AKRASH-Nadeem** [community/community#131062] [Mobile]
Hi,

I was adding a label to my repo issues,I wanted to create a new label but I don't seem to find the option to add one.

GitHub mobile version: 1.167.1

**Comment by Rishi-Sudhakar** [comment]
I believe you probably can't do it in mobile. If you still want to do it in mobile, use a browser> use desktop view> go to github website> in issues tab> label> create a new label.

**Reply by AKRASH-Nadeem** [reply]
using mobile app for that could be a time saving process because i don't want to use browser for my github so i have to login and logout again and again.
i have password set on my GitHub application to prevent others from doing anything by mistake. But i can't login in browser so i have to set password on browser too which is not a good idea.

**Comment by howprice** [comment]
I find the lack of this feature on mobile very inconvenient. I use the Android app "on-the-go" when I am away from my desktop to orgainise work. The web interface is extremely fiddly to use in a mobile web browser.

Is it possible to escalate this discussion into an issue?
